Atomic/rpm_host_install.py

```diff
@@ -36,10 +36,24 @@ def _copyfile(selinux_hnd, src, dest):
             os.symlink(linkto, dest)
             return True
         else:
-            # we cannot use shutil.copy2() or shutil.copystat() here as it would override the
-            # security.selinux xattr.
-            shutil.copy(src, dest)
-            shutil.copymode(src, dest)
+            file_copied = False
+            # newer version of Skopeo/SELinux use the same SELinux context for f
```

Comment:
Incomplete comment? maybe f refer to file here?

Reply:
fixed in a fixup patch